src/schema/schema_drop.c

```diff
@@ -74,13 +73,13 @@ __drop_colgroup(WT_SESSION_IMPL *session, const char *uri, bool force, const cha
  *     WT_SESSION::drop for an index.
  */
 static int
-__drop_index(WT_SESSION_IMPL *session, const char *uri, bool force, const char *cfg[])
+__drop_index(WT_SESSION_IMPL *session, const char *uri, const char *cfg[])
 {
     WT_DECL_RET;
     WT_INDEX *idx;
 
     /* If we can get the index, detach it from the table. */
-    if ((ret = __wt_schema_get_index(session, uri, true, force, &idx)) == 0)
+    if ((ret = __wt_schema_get_index(session, uri, true, false, &idx)) == 0)
```

Comment:
This change is not really necessary although it doesn't matter since removing indexes is part of PM-3441. Since the arg to `wt_schema_get_index` is to be quiet instead of print a message, I would suggest leaving this as-is so that we don't get spurious messages and possible failures until the functionality is removed.

Otherwise, there is no caller of `wt_schema_get_index` with `true` so the arg could be removed entirely.

Reply:
This was done since force was removed from __drop_index().